readme.md

```diff
@@ -707,6 +707,7 @@
 - [Biomedical Information Extraction](https://github.com/caufieldjh/awesome-bioie#readme) - How to extract information from unstructured biomedical data and text.
 - [Web Archiving](https://github.com/iipc/awesome-web-archiving#readme) - An effort to preserve the Web for future generations.
 - [WP-CLI](https://github.com/schlessera/awesome-wp-cli#readme) - Command-line interface for WordPress.
+- [QR codes](https://github.com/aureooms/awesome-qr-code#readme) - QR code is the trademark for a type of matrix barcode.
```

Comment:
The description could be improved. The fact that it's a trademark is completely uninteresting in this context. I think it should also better explain exactly what a QR code is.

Reply:
Also, title-case on the title.

Reply:
Thanks! I just tried to fix that in 4f4efca and 7c17b10.